src/test/java/org/springframework/data/elasticsearch/core/DefaultResultMapperTests.java

```diff
@@ -152,6 +158,69 @@ public void setsIdentifierOnImmutableType() {
 		assertThat(result.getId(), is("identifier"));
 	}
 
+	@Test // DATAES-198
```

Comment:
minor, but could we move the comment above `@Test`

Reply:
This is explicitly called out in the contrib. guidelines:

> In the Javadoc for the newly added test cases refer to the ticket the changes apply to like this

```
@Test // DATAES-...
```

Reply:
Ohh, sorry for that ;)